File: Assignment1/FloatMultiplication.cpp
#include<iostream>
using namespace std;	
class FloatMultilication
{
	public:
	double num1,num2,mul;
	void getdata()
	{
		cout<<"Enter first number:"<<endl;
	    cin>>num1;
	    cout<<"Enter second number:"<<endl;
	    cin>>num2;
	}
	void cal_multiplication()
	{
		mul=num1*num2;         
	}
	void display()
	{
		cout<<"----------------------"<<endl;
		cout<<"multiplication:"<<mul<<endl;
		cout<<"----------------------"<<endl;
	}
};
int main() 
{
	FloatMultilication fm;
	fm.getdata();
	fm.cal_multiplication();
	fm.display();

}

File: JAVA/Assignment1/FloatMultiplication.java
/******10.WAP to Multiply two Floating Point Numbers****/
import java.util.*;
class FloatMultiplication
{
float num1,num2;
void getdata()
{
Scanner sc=new Scanner(System.in); // help to take i/p from user
System.out.println("Enter number");
num1=sc.nextInt();
System.out.println("Enter number");
num2=sc.nextInt();
}
void multiply()
{
num1=num1*num2;
}
void display()
{
System.out.println("-----------------");
System.out.println("Multiplication:"+num1);
System.out.println("-----------------");
}
public static void main(String[] args)
{
FloatMultiplicationm1=new FloatMultiplication();
fm.getdata();
fm.multiply();
fm.display();
}
}